autoload/neoterm/repl.vim

```diff
@@ -59,7 +59,9 @@ endfunction
 function! g:neoterm.repl.exec(command)
   if !l:self.loaded && !g:neoterm_direct_open_repl
     if !empty(get(g:, 'neoterm_repl_command', ''))
-      call l:self.instance().do(g:neoterm_repl_command)
+      if !exists('g:neoterm_no_auto_repl_cmd')
```

Comment:
instead of checking if the variable exists here, would be better to set a default value for it on `plugin/neoterm.vim` and just check the value of the variable here.